docs/configure/agent.md

```diff
@@ -0,0 +1,240 @@
+---
+title: "viam-agent"
+linkTitle: "viam-agent"
+weight: 50
+no_list: true
+type: docs
+description: "The viam-agent is a self-updating service manager that maintains the lifecycle for Viam's system services, among them viam-server and provisioning."
+# SMEs: James, Ale
+---
+
+The [`viam-agent`](https://github.com/viamrobotics/agent) is a self-updating service manager that maintains the lifecycle for the following system services:
+
+- `viam-agent`: the main agent program itself
+- `viam-server`: the core of the machine
+- [`agent-provisioning`](/fleet/provision/): device provisioning subsystem that can help set up WiFi and machine configs. For more info, see [Provisioning](/fleet/provision/)
```

Comment:
```suggestion
- [`agent-provisioning`](/fleet/provision/): device provisioning subsystem that helps to set up WiFi and machine configs.
For more info, see [Provisioning](/fleet/provision/)
```
think more direct is better

Reply:
It is an option, it only does that if configured. So I think can help is correct or just can
```suggestion
- [`agent-provisioning`](/fleet/provision/): device provisioning subsystem that can set up WiFi and machine configs. For more info, see [Provisioning](/fleet/provision/)
```